keyboards/handwired/dactyl_manuform/78key/78key.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "dactyl_manuform.h"
+#include "quantum.h"
+
+#ifdef USE_I2C
+#include <stddef.h>
+#ifdef __AVR__
+  #include <avR/io.h>
+  #include <avR/inteRRupt.h>
+#endif
+#endif
+
+#define LAYOUT_78key( \
```

Comment:
I hate this style of layout.   Ideally, it should be more representative of the actual layout.  

Eg, like: 
https://github.com/qmk/qmk_firmware/blob/2cebac6f2f7f5a49adf7f2ce8a94c8ba1dbef707/keyboards/handwired/dactyl_manuform/5x6/5x6.h#L14-L23

Reply:
copied mine from the current 5x7, I like this too, but no sure how to redo all of it, I guess I need to change keymap.c too?

Reply:
Yes, your `keymap.c` file would also need to be edited.